coop_badge_reader/__init__.py

```diff
@@ -1,2 +1,3 @@
-# -*- coding: utf-8 -*-
+# Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
+
```

Comment:
No need to add copyright on init files :) 
it's ok though, no need to remove it either